src/eval.c

```diff
@@ -6138,6 +6138,58 @@ class_tv2string(typval_T *tv, char_u **tofree)
     return r;
 }
 
+/*
+ * Return a textual representation of an Object in "tv".
+ * If the memory is allocated "tofree" is set to it, otherwise NULL.
+ * When "copyID" is not zero replace recursive object with "...".
+ * When "restore_copyID" is FALSE, repeated items in the object are
+ * replaced with "...".  May return NULL.
+ */
+    static char_u *
+object_tv2string(
+    typval_T	*tv,
+    char_u	**tofree,
+    int		copyID,
+    int		restore_copyID,
+    char_u	*numbuf,
+    int		echo_style,
+    int		composite_val)
+{
+    char_u	*r = NULL;
+
+    object_T	*obj = tv->vval.v_object;
+    if (obj == NULL || obj->obj_class == NULL)
+    {
+	*tofree = NULL;
+	r = (char_u *)"object of [unknown]";
+    }
+    else if (copyID != 0 && obj->obj_copyID == copyID
+            && obj->obj_class->class_obj_member_count != 0)
+    {
+	int n = 25 + strlen((char*)obj->obj_class->class_name);
```

Comment:
This should be a `size_t` rather than an `int`, MSVC emits a warning otherwise.
Nit: `char*` should be `char *`, ditto for the other one in the `snprintf` below.

Reply:
> This should be a `size_t` rather than an `int`, MSVC emits a warning otherwise. Nit: `char*` should be `char *`, ditto for the other one in the `snprintf` below.

Thanks, #15187